### PR TITLE
Changes to LRF30M90HS, LRF20M3PS, and LRF50M3PS

### PR DESCRIPTION
**Pull request by ferrowism**
Markdown added for LRF30M90HS and LRF20M3PS
Python change for LRF20M3PS

### CODE REVIEW

**Comment by ferrowism** [comment]
Also changed style for LRF30M90HS Arduino sample code, given Arduino/C is more complex than python, added more explanation comments

**Comment by ferrowism** [comment]
Also added "20 meter" to LRF20M3PS markdown file to more clearly differentiate it LRF50M3PS markdown file

File: LRF20M3PS/LRF20M3PS-samplecode.py [hunk 1]
```diff
@@ -3,67 +3,71 @@
...
-ser = serial.Serial()
-#ser.port = '/dev/ttyUSB0' #(Linux)Note usb port should reflect which port the module is plugged in to - use ls /dev/ttyUSB* command to identify port
-ser.port = 'COM4'          #(Windows)Note usb port should reflect which port the module is plugged in to COM port, usw device manager to identify port
-ser.baudrate = 115200
-ser.bytesize = 8
-ser.parity = 'N'
-ser.topbits = 1
-ser.timeout = 3000
-ser.open()
+#List available COMport
```
Comment (RacingLiu) [inline LRF20M3PS/LRF20M3PS-samplecode.py:9]: 需要確認這部分在Linux可不可以運作

**Review by RacingLiu — COMMENTED** [review]
需要確認這部分在Linux可不可以運作

File: LRF20M3PS/LRF20M3PS-samplecode.py (hunk 1 above)
Comment (ferrowism) [reply LRF20M3PS/LRF20M3PS-samplecode.py:9]: 確認了，在Linux可以運作

**Review by RacingLiu — APPROVED** [review]

**Review by RacingLiu — APPROVED** [review]

**Pull request merged**